xbmc/pvr/PVRManager.h

```diff
@@ -72,8 +72,7 @@ namespace PVR
   enum ContinueLastChannelOnStartup
   {
     CONTINUE_LAST_CHANNEL_OFF  = 0,
-    CONTINUE_LAST_CHANNEL_IN_BACKGROUND,
-    CONTINUE_LAST_CHANNEL_IN_FOREGROUND
+    CONTINUE_LAST_CHANNEL_IN_BACKGROUND
```

Comment:
Perhaps, but the value isn't used anywhere in the actual code. It's basically "if not foreground", not explicitly "is background". Not sure if we should leave this here or not.